android/java/org/chromium/chrome/browser/vpn/BraveVpnPrefUtils.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.vpn;
+
+import android.content.SharedPreferences;
+
+import org.chromium.base.ContextUtils;
+import org.chromium.chrome.R;
+
+public class BraveVpnPrefUtils {
+    private static final String PREF_BRAVE_VPN_CALLOUT = "brave_vpn_callout";
+    private static final String PREF_BRAVE_VPN_CALLOUT_SETTINGS = "brave_vpn_callout_settings";
+    private static final String PREF_BRAVE_VPN_SUBSCRIPTION_PURCHASE =
+            "brave_vpn_subscription_purchase";
+    private static final String PREF_BRAVE_VPN_HOSTNAME = "brave_vpn_hostname";
+    private static final String PREF_BRAVE_VPN_PURCHASE_TOKEN = "brave_vpn_purchase_token";
+    private static final String PREF_BRAVE_VPN_PRODUCT_ID = "brave_vpn_product_id";
+    private static final String PREF_BRAVE_VPN_PURCHASE_EXPIRY = "brave_vpn_purchase_expiry";
+    private static final String PREF_BRAVE_VPN_SERVER_REGIONS = "brave_vpn_server_regions";
+    private static final String PREF_BRAVE_VPN_SERVER_CHANGE_LOCATION = "server_change_location";
+
+    public static final String PREF_BRAVE_VPN_AUTOMATIC = "automatic";
+    public static final String PREF_BRAVE_VPN_FEATURE = "brave_vpn_feature";
+
+    private static final SharedPreferences mSharedPreferences =
+            ContextUtils.getAppSharedPreferences();
+
+    public static boolean isBraveVpnFeatureEnable() {
```

Comment:
maybe change names `isBraveVpnFeatureEnabled`/`setBraveVpnFeatureEnabled` , like with meaning of the complete action?